apps/src/code-studio/pd/teacher_application/teacher_application.jsx

```diff
@@ -25,6 +25,11 @@ function FieldGroup({ id, label, validationState, required, errorText, ...props
   if (errorText) {
     validationState = 'error';
   }
+
+  if (!(props['maxLength'])) {
+    props['maxLength'] = '1000';
```

Comment:
it seems like this needs to be a number for the propType check below to succeed. what am I missing?